docs/broker.rst

```diff
@@ -177,8 +177,8 @@ your management. The method takes an optional parameter ``search_parameters`` wh
     # search for accounts created after January 30th 2022.
     #Response should contain Contact and Identity fields for each account.
     filter = ListAccountsRequest(
-                        created_after="2022-01-30",
-                        entities=[AccountEntities.Contact, AccountEntities.Identity]
+                        created_after=datetime.datetime.strptime("2022-01-30", "%Y-%m-%d"),
```

Comment:
Hmm this should've worked with strings 🤔. In any case, let's leave it this way for now.